### DISCUSSION TITLE
can not clean up old version software. Cannot remove item xxx: Access to the path

### DISCUSSION THREAD

**Discussion by xianzheTM** [ScoopInstaller/Scoop#5456] [Help and Questions]
#### Current Behavior
```
❯ scoop cleanup *
Removing argocd: 2.6.5
Removing GitKraken: 9.2.0
Removing groovy: 4.0.9
Removing helm: 3.11.1
Removing kubectl: 1.26.2
Removing kustomize: 5.0.0
Removing msys2: 2023-01-27Remove-Item: E:\UserScoop\apps\scoop\current\libexec\scoop-cleanup.ps1:50
Line |
  50 |          Remove-Item $dir -ErrorAction Stop -Recurse -Force
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot remove item E:\UserScoop\apps\msys2\2023-01-27\home.original\home: Access to the path
     | 'E:\UserScoop\apps\msys2\2023-01-27\home.original\home' is denied.
```

#### Expected Behavior
Clean up all old versions of software.

#### Additional context/output
After manually deleting E:\UserScoop\apps\msys2\2023-01-27\home.original, the scoop cleanup command was executed successfully.
It's not just msys2 that has this problem, it's also happened when uninstalling other software.

### System details

**Windows version:** 11

**OS architecture:**  64bit

**PowerShell version:** 
```
❯ $PSVersionTable.PSVersion

Major  Minor  Patch  PreReleaseLabel BuildLabel
-----  -----  -----  --------------- ----------
7      3      3
```

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
  "last_update": "2023-03-19T19:53:44.5845705+08:00",
  "proxy": "localhost:7890",
  "aria2-enabled": true,
  "scoop_repo": "https://github.com/ScoopInstaller/Scoop",
  "scoop_branch": "master"
}

```

**Comment by rashil2000** [comment]
How did you delete  E:\UserScoop\apps\msys2\2023-01-27\home.original? Were there any errors?

**Reply by xianzheTM** [reply]
Delete in file explorer.There are no errors, it's like deleting a normal file.

**Reply by rashil2000** [reply]
Are you able to reproduce the error again, with some different package perhaps?

**Reply by xianzheTM** [reply]
I have no way to reproduce this problem. I've encountered this problem a few times, each time with a different app. The trouble is that I have no way to control the application updates that make sense to execute the `scoop cleanup *` and have to wait.

**Reply by rashil2000** [reply]
Hmm, I see. I'll have to check if I encounter it to diagnose it properly.
